### PR TITLE
feat(orchestrator): use v2 endpoints to retrieve workflow overviews

### PR DESCRIPTION
**Pull request by gciavarrini**
Use v2 endpoints in orchestrator plugin UI for workflow overview list and workflow overview by id.

Warning: This **DELETE** corresponding v1 endpoints.

### CODE REVIEW

File: plugins/orchestrator/src/components/WorkflowInstancePage.stories.tsx [hunk 1]
```diff
@@ -1,156 +1,156 @@
...
-  },
-};
-
-export const AutoRefresh3Errors: Story = {
-  name: 'Auto refresh three errors',
-  args: {
-    instanceId: '__auto_refresh_three_errors__',
-  },
-};
+// import React from 'react';
```
Comment (batzionb) [inline plugins/orchestrator/src/components/WorkflowInstancePage.stories.tsx:1]: @gciavarrini 
Best just delete the stories files 
We have them in the history

File: plugins/orchestrator/src/api/OrchestratorClient.ts [hunk 2]
```diff
@@ -78,11 +104,24 @@ export class OrchestratorClient implements OrchestratorApi {
...
+    try {
+      return await defaultApi.getWorkflowsOverview(
+        paginationInfo?.page,
+        paginationInfo?.pageSize,
+        paginationInfo?.orderBy,
+        paginationInfo?.orderDirection,
+        reqConfigOption,
+      );
+    } catch (error: any) {
+      throw new Error(error);
```
Comment (batzionb) [inline plugins/orchestrator/src/api/OrchestratorClient.ts:123]: no point in just catching an error and throwing it again as is...

File: plugins/orchestrator/src/components/WorkflowInstancePage.stories.tsx (hunk 1 above)
Comment (gciavarrini) [reply plugins/orchestrator/src/components/WorkflowInstancePage.stories.tsx]: Done

File: plugins/orchestrator/src/api/OrchestratorClient.ts (hunk 2 above)
Comment (gciavarrini) [reply plugins/orchestrator/src/api/OrchestratorClient.ts]: addressed

File: plugins/orchestrator/package.json [hunk 3]
```diff
@@ -72,7 +72,8 @@
     "react-use": "^17.4.0",
     "swr": "^2.0.0",
     "uuid": "^9.0.1",
-    "vscode-languageserver-types": "^3.16.0"
+    "vscode-languageserver-types": "^3.16.0",
+    "axios": "1.6.8"
```
Comment (batzionb) [inline plugins/orchestrator/package.json:76]: This is just for the tests?
Add it to devDependencies

Comment (gciavarrini) [reply plugins/orchestrator/package.json:76]: No it's needed in the code too see https://github.com/janus-idp/backstage-plugins/blob/9e4e66c264b1fe31d91498e124d73670c10f70c6/plugins/orchestrator/src/api/OrchestratorClient.ts#L5-L9

Comment (batzionb) [reply plugins/orchestrator/package.json:76]: right, my mistake

**Review by batzionb — APPROVED** [review]

**Pull request merged**